src/EnergyPlus/CommandLineInterface.cc

```diff
@@ -234,7 +234,7 @@ Built on Platform: {}
         // bool debugCLI = false;
         app.add_flag("--debug-cli", debugCLI, "Print the result of the CLI assignments to the console and exit")->group(""); // Empty group to hide it
 
-#if LINK_WITH_PYTHON
+#if LINK_WITH_PYTHON && PYTHON_CLI
```

Comment:
Only build out the auxiliary CLI if the new CLI flag is also enabled.

Reply:
(side note, but shouldn't this be #ifdef ?)

Reply:
That sounds fine.  I changed it to ifdef and retested everything.